.openshift-ci/google-cloud-sdk/install.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
I see that part of this script is inspired by https://github.com/CircleCI-Public/gcp-cli-orb. Can we add a comment referencing it so we know where to look if we run into trouble in the future?

Reply:
Yep, a good point. I've stripped some bits from there, but it's definitely heavily inspired by the orb.